env-control

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Wrapper to control creation/deletion of Conda Environments within JupyterLab hosted on Datalabs
```

Comment:
This is pretty neat!

Reply:
I second that!

Reply:
This is Iain's work that I copied from the Lab container